tests/ResourceFormatTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use Dedoc\Scramble\PhpDoc\PhpDocParser;
+use PHPStan\PhpDocParser\Lexer\Lexer;
+use PHPStan\PhpDocParser\Parser\ConstExprParser;
+use PHPStan\PhpDocParser\Parser\TokenIterator;
+use PHPStan\PhpDocParser\Parser\TypeParser;
+
+function getFormatFromDoc(string $phpDoc)
+{
+    $lexer = new Lexer();
+    $tokenized = $lexer->tokenize($phpDoc);
+
+    $constExprParser = new ConstExprParser();
+    $typeParser = new TypeParser($constExprParser);
+    $phpDocParser = new PhpDocParser($typeParser, $constExprParser);
+
+    $tokens = new TokenIterator($tokenized);
+
+    $node = $phpDocParser->parse($tokens);
+    $withFormat = $node->getTagsByName("@format");
+    if(count($withFormat) === 0) return null;
+
+    $withFormat = reset($withFormat);
+
+    return $withFormat->value->value;
+}
+
+it('handles string type with format', function ($phpDoc, $expectedFormat) {
+    $result = getFormatFromDoc($phpDoc);
+
+    expect($result)->toBe($expectedFormat);
+})->with([
+    ["/** @var string \n * @format email */", "email"],
```

Comment:
After closer review of this test, the test is not really testing newly added extension's code.